### ISSUE TITLE
sitewide, forms — 508-defect-3 [FOCUS MANAGEMENT]: Focus outline SHOULD be removed from non-linked heading

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#9116]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

> Located in [VSA BAM2 MDOT MVP a11y spot check](https://github.com/department-of-veterans-affairs/va.gov-team/issues/5868#issuecomment-627475702)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Focus outline **should** not be applied to non-interactive focusable content, as a pattern on VA.gov.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a keyboard user, I want to identify interactive focusable items by distinctive focus styles.

## Environment

Operating System: all
Browser: all
Server destination: staging

## Steps to Recreate

1. Enter https://staging.va.gov/hearing-aid-batteries-and-accessories/introduction in browser
2. Log in as username: vets.gov.user+1@gmail.com 
3. Start screenreading device listed in Environment
3. Move through the process flow, noting that the focus outline appears on the non-linked headings as shown in the screenshot

## Possible Fixes (optional)

See [#457](https://github.com/department-of-veterans-affairs/va.gov-team/issues/457)

## WCAG or Vendor Guidance (optional)

* [W3C Understanding WCAG 2.0 - Consistent Identification SC 3.2.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-consistent-functionality.html)

## Screenshots or Trace Logs

![Screen Shot 2020-05-12 at 1 10 26 PM](https://user-images.githubusercontent.com/57469/81724214-fd3b3880-9451-11ea-9b1d-bb64debc7ff7.png)

**Comment by k80bowman** [comment]
@CourtneyChambers77 could you please follow the Steps to Recreate and check if this is still an issue?

**Comment by bkjohnson** [comment]
Applications are [doing this in their custom CSS](https://github.com/department-of-veterans-affairs/vets-website/blob/a53302b9eb20c002aa662c05232821783a38aa6b/src/applications/debt-letters/sass/debt-letters.scss#L47-L49), so we could help simplify app CSS if this is something we wanted to implement at the design system level.

**Comment by CourtneyChambers77** [comment]
Tested this focus element and it no longer has an issue
